### PR TITLE
Adds Channels

### PR DESCRIPTION
**Pull request by pschoffer**
We are adding `release_channel` (staging/prod) to changelog posts for mu-plugins.

### CODE REVIEW

File: scripts/github-changelog.php [hunk 1]
```diff
@@ -136,6 +138,10 @@ function create_draft_changelog( $title, $content, $tags ) {
         'tags' => implode( ',', $tags ),
     ];
 
+    if ( $channels ) {
```
Comment (brunobasto) [inline scripts/github-changelog.php:141]: Since this is an array, should we check for array length instead?

**Review by brunobasto — APPROVED** [review]
Left a minor comment, but LGTM 

Thank you @pschoffer

File: scripts/github-changelog.php (hunk 1 above)
Comment (pschoffer) [reply scripts/github-changelog.php:141]: We could but empty array is falsy in `php`. So I think this is fine (I also tested that it doesn't add the new field if you don't send in the parameter).

Comment (brunobasto) [reply scripts/github-changelog.php:141]: > empty array is falsy in php

Nice!

**Pull request merged**